addons/sys_io/fnc_ts3ChannelMove.sqf

```diff
@@ -15,6 +15,10 @@
  * Public: No
  */
 
-private _ts3ChannelDetails = format ["%1,%2,%3", EGVAR(sys_core,ts3ChannelName), EGVAR(sys_core,ts3ChannelPassword), serverName];
+private _serverName = toString ((toArray serverName) select {_x < 127});
+if (_serverName == "") then {
+    _serverName = "ServerName Fully Sanitized away";
```

Comment:
Can we add a warning for RPT for that as well?

Reply:
done